README.md

```diff
@@ -254,12 +260,14 @@ You can also choose to use the extend functionality to add an `$auto` and
 `$autoArray` command:
 
 ```js
-update.extend('$auto', function(value, object) {
+import { defaultContext as context, update } from 'immutability-helper';
```

Comment:
#### Breaking change
No more default exports, so every import is in the format `import { x, y, z } from 'immutability-helper';`

Reply:
What if we keep the default export and re-export it as `update`?
Users can always do `import update, { defaultContext as context } from 'immutability-helper'` if they want to.

Reply:
See 4fc6303

1. I am no longer exporting `defaultContext`, because I am now exporting its `update` and `extend` methods, bound to the default context (as the library already does).
1. I removed support for `update.extend` for two reasons:
   - Two ways of doing the same thing is generally confusing.
   - It's hard, if not impossible to add properties to a method. I can do it with an interface and have `Context implements IUpdate`, but it doesn't compile that way.
1. I removed any reference to `require('immutability-helper').default` in the README because we shouldn't advertise this as a golden path. Let's call it an undocumented feature.